### ISSUE TITLE
msvc internal compiler error

### ISSUE THREAD

**Issue by xoviat** [wxWidgets/Phoenix#348]
```
     Creating library siplib.lib and object siplib.exp
  [ 79/764] cxx: sip\cpp\sip_corewxContextMenuEvent.cpp -> build\waf\3.6\x64\release\sip\cpp\sip_corewxContextMenuEvent.
cpp.3.o
  [ 80/764] cxx: sip\cpp\sip_corewxControl.cpp -> build\waf\3.6\x64\release\sip\cpp\sip_corewxControl.cpp.3.o
  [ 81/764] cxx: sip\cpp\sip_corewxControlWithItems.cpp -> build\waf\3.6\x64\release\sip\cpp\sip_corewxControlWithItems.
cpp.3.o
  [ 82/764] cxx: sip\cpp\sip_corewxCursor.cpp -> build\waf\3.6\x64\release\sip\cpp\sip_corewxCursor.cpp.3.o
  sip_corewxCommandList.cpp
  c:\users\User\appdata\local\temp\pip-build-3y5mq8t9\wx\sip\cpp\sip_corewxcommandlist.cpp(227): fatal error C1001:
An internal error has occurred in the compiler.
  (compiler file 'msc1.cpp', line 1468)
   To work around this problem, try simplifying or changing the program near the locations listed above.
  Please choose the Technical Support command on the Visual C++
   Help menu, or open the Technical Support help file for more information
  Internal Compiler Error in C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\CL.exe.  You will be promp
ted to send an error report to Microsoft later.
  INTERNAL COMPILER ERROR in 'C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\CL.exe'
      Please choose the Technical Support command on the Visual C++
      Help menu, or open the Technical Support help file for more information
  sip_corewxContextHelp.cpp
  c:\users\User\appdata\local\temp\pip-build-3y5mq8t9\wx\sip\cpp\sip_corewxcontexthelp.cpp(142): fatal error C1001:
An internal error has occurred in the compiler.
  (compiler file 'msc1.cpp', line 1468)
   To work around this problem, try simplifying or changing the program near the locations listed above.
  Please choose the Technical Support command on the Visual C++
   Help menu, or open the Technical Support help file for more information
  Internal Compiler Error in C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\CL.exe.  You will be promp
ted to send an error report to Microsoft later.
  INTERNAL COMPILER ERROR in 'C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\CL.exe'
      Please choose the Technical Support command on the Visual C++
      Help menu, or open the Technical Support help file for more information
  sip_corewxContextHelpButton.cpp
  sip_corewxContextMenuEvent.cpp
  sip_corewxControl.cpp
  sip_corewxControlWithItems.cpp
  sip_corewxCursor.cpp
  c:\users\User\appdata\local\temp\pip-build-3y5mq8t9\wx\sip\cpp\sip_corewxcursor.cpp(24): warning C4311: 'type cast
': pointer truncation from 'WXHANDLE' to 'long'
  c:\users\User\appdata\local\temp\pip-build-3y5mq8t9\wx\sip\cpp\sip_corewxcursor.cpp(24): warning C4302: 'type cast
': truncation from 'WXHANDLE' to 'long'
  c:\users\User\appdata\local\temp\pip-build-3y5mq8t9\wx\sip\cpp\sip_corewxcursor.cpp(32): warning C4312: 'type cast
': conversion from 'long' to 'WXHANDLE' of greater size
  Waf: Leaving directory `C:\Users\User\AppData\Local\Temp\pip-build-3y5mq8t9\wx\build\waf\3.6\x64\release'
  Build failed
   -> task in '_core' failed (exit status 2):
        {task 2256018371864: cxx sip_corewxCommandList.cpp -> sip_corewxCommandList.cpp.3.o}
  ['C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\amd64\\CL.exe', '/nologo', '/UNDEBUG', '/Gy', '/EHsc
```

**Comment by RobinD42** [comment]
I haven't seen one of those in a long while...  Some things to check:

 * Ensure that your Visual Studio is up to date
 * Ensure you don't have too much other stuff running on the system when building
 * You could try using the `--jobs=1` flag on the `build.py` command line, so it will not try to run more than one compile task at the same time.

**Comment by xoviat** [comment]
msvc installed using 2015 build tools and web installer claims that it's the latest version. 2017 is currently in preview, so I wouldn't install that.

**Comment by xoviat** [comment]
let me try again

**Comment by xoviat** [comment]
I received the same error when attempting to compile again, but honestly I don't think your time would be well-spent trying to fix this.

**Comment by RobinD42** [comment]
Ok. If you do figure it out please let us know the cause of the issue in case anybody else runs into it.
